roles/neutronstar/defaults/main.yml

```diff
@@ -4,8 +4,13 @@ neutronstar_dest: "{{ sw_path }}/neutronstar"
 # neutronstar_container_url: "docker://nf-core/neutronstar:{{ rnaseq_version }}"
```

Comment:
Sorry, `rnaseq_version`?

Reply:
Ah, that line is commented out and won't run. It was leftover from before but I can remove it if you want.